trade-finance-manager-api/src/v1/mappings/premium-schedule/index.js

```diff
@@ -23,11 +24,11 @@ const mapPremiumScheduleFacility = (facility, facilityExposurePeriod, facilityGu
 
   mapped.dayBasis = String(facility.dayCountBasis);
   mapped.exposurePeriod = facilityExposurePeriod;
-  mapped.maximumLiability = facility.ukefExposure;
+  mapped.maximumLiability = Number(stripCommas(facility.ukefExposure));
 
   mapped.cumulativeAmount = 0;
   if (facility.disbursementAmount) {
-    mapped.cumulativeAmount = facility.disbursementAmount;
+    mapped.cumulativeAmount = Number(stripCommas(facility.disbursementAmount));
```

Comment:
Can we add a test for the changes in this file, to make sure these fields have their commas stripped and are changed to a number?